src/components/cards/employees-card/EmployeeCard.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div class="employee-card">
+
+      <h4 class="employeeCardName">{{employeeFirstName}}</h4>
+      <h4 class="employeeCardLastName">{{employeeLastName}}</h4>
```

Comment:
Also it's better to put a space after {{ and before }}
eg: `{{ employeeLastName }}` instead `{{employeeLastName}}`